### PR TITLE
Merge Duplicate Guest Entries

### LINKED ISSUE #521: Multiple Guest Entries for 1 Person
### URL

https://www.jupiterbroadcasting.com/guests/page/2/
~https://www.jupiterbroadcasting.com/guests/page/4/~
https://www.jupiterbroadcasting.com/guests/page/6/

### On what type of device(s) did you see this bug?

Phone, Tablet, Desktop

### On which browser(s) are you seeing the problem?

Firefox, Chrome, Safari, Microsoft Edge

### Other Browser

_No response_

### Summary

I am noticing duplicate entries for guests on the show. These should probably be merged.

These are all the potentially duplicate guests I have seen:
- [cat](https://www.jupiterbroadcasting.com/guests/cat) and [catherine-kretzschmar](https://www.jupiterbroadcasting.com/guests/catherine-kretzschmar/)
- ~[heather](https://www.jupiterbroadcasting.com/guests/heather) and [heather-ellsworth](https://www.jupiterbroadcasting.com/guests/heather-ellsworth/)~
- [jeromelaban](https://www.jupiterbroadcasting.com/guests/jeromelaban) and [jlaban](https://www.jupiterbroadcasting.com/guests/jlaban)

### PR DESCRIPTION
**Pull request by 1701home**
Merged people's information and updated show notes to use the new username. Removed old images. I hope it closes #521

### CODE REVIEW

File: content/people/catherine-kretzschmar.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-avatar: /images/people/catherine-kretzschmar.jpg
-avatar_small: /images/people/catherine-kretzschmar_small.jpg
+avatar: /images/people/cat.jpg
+avatar_small: /images/people/cat_small.jpg
```
Comment (CGBassPlayer) [inline content/people/catherine-kretzschmar.md:3]: For the sake of consistency I would rename `cat.jpg` and `cat_small.jpg` to `catherine-kretzschmar.jpg` and `catherine-kretzschmar_small.jpg` just so they remain consistent with the username

**Review by CGBassPlayer — CHANGES_REQUESTED** [review]
Overall, it looks really good! Just a small nit pick change and this should be ready to merge

File: content/people/catherine-kretzschmar.md (hunk 1 above)
Comment (1701home) [reply content/people/catherine-kretzschmar.md:3]: That is an excellent point, will do.

**Review by CGBassPlayer — APPROVED** [review]
LGTM

**Pull request merged**